<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>by:demorz</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_net">
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string>选择网络</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="indent">
         <number>36</number>
        </property>
        <property name="buddy">
         <cstring></cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="netchoose">
        <item>
         <property name="text">
          <string>Resnet18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Resnet34</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Resnet50</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="oldnet">
        <property name="text">
         <string>读取保存的网络参数</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="newnet">
        <property name="text">
         <string>新建网络</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="l_p_lab">
        <property name="text">
         <string>从...读取</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ck_path">
        <property name="text">
         <string>.\checkpoint\ckpt.pth</string>
        </property>
        <property name="placeholderText">
         <string>保存的参数地址...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="getpath">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>选择优化器</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="optimizer">
        <item>
         <property name="text">
          <string>SGD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RMSProp</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Adam</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Adamax</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SparseAdam</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_LR">
        <property name="text">
         <string>初始学习率</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="LR">
        <property name="specialValueText">
         <string>0.1</string>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_mom">
        <property name="text">
         <string>动量因子</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="Mom">
        <property name="specialValueText">
         <string>0.9</string>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.900000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_gam">
        <property name="text">
         <string>gamma学习率衰减</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="gam">
        <property name="specialValueText">
         <string>0.9</string>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.900000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_L2">
        <property name="text">
         <string>权重衰减</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="L2">
        <property name="specialValueText">
         <string>0.0005</string>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
        <property name="value">
         <double>0.000500000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="num_lab">
        <property name="text">
         <string>每多少轮保存</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="num">
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_num_lab">
        <property name="text">
         <string>最大训练轮数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="max_num">
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>保存到...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ck_path_s">
        <property name="text">
         <string>.\checkpoint</string>
        </property>
        <property name="placeholderText">
         <string>参数保存到...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ckpt_s_p_b">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>文件名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="s_name">
        <property name="text">
         <string>ckpt</string>
        </property>
        <property name="placeholderText">
         <string>保存的文件名</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QRadioButton" name="cf10">
        <property name="text">
         <string>cifar10</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="cf100">
        <property name="text">
         <string>cifar100</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup2</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>batchsize</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="ba_size">
        <property name="maximum">
         <number>2048</number>
        </property>
        <property name="value">
         <number>128</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="create_net">
      <property name="text">
       <string>生成</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup2"/>
 </buttongroups>
</ui>
